pybossa/sched.py

```diff
@@ -535,6 +535,29 @@ def get_task_users_key(task_id):
 def get_user_tasks_key(user_id):
     return USER_TASKS_KEY_PREFIX.format(user_id)
 
+def get_locked_tasks_project(project_id):
+    tasks = []
+    redis_conn = sentinel.master
+
+    # Get the active users key for this project.
+    key = get_active_user_key(project_id)
+
+    # Get the users for each locked task.
+    for user_key in redis_conn.hgetall(key).iteritems():
+        user_id = user_key[0]
+
+        # Get the tasks locked by this user.
+        task_id, seconds_remaining = get_task_id_and_duration_for_project_user(project_id, user_id)
```

Comment:
`get_task_id_and_duration_for_project_user` returns the task with maximum remaining timeout. Since we allow users to lock multiple tasks in Task Queue scheduler, we need to return all unexpired tasks locked by this user. 

We can use this function to get all tasks locked by user from Redis, may need to filter out those already expired. https://github.com/bloomberg/pybossa/blob/f847c0901c4dc9f51c3258525c4ce40a82d7ffad/pybossa/sched.py#L513

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/615/commits/8b142b53a875f6e5f4e1c5fcb8bd602d72d6affb